src/main/main.ts

```diff
@@ -339,7 +358,7 @@ ipcMain.on(ChannelEnum.LOAD_AUTO_EQ_PRESET, async (event, arg) => {
   const [deviceName, responseName] = arg;
 
   try {
-    const presetSettings: IPreset = getAutoEqPreset(deviceName, responseName);
+    const presetSettings: IPresetV2 = getAutoEqPreset(deviceName, responseName);
     state.preAmp = presetSettings.preAmp;
     state.filters = presetSettings.filters;
```

Comment:
I feel like we shouldnt to worry about user presets. I believe if users tamper with the preset file, its out of our scope to manage that. For autoeq, I think it would be fair to check for filter length > 0 in the test.